### ISSUE TITLE
Cannot create floating ip

### ISSUE THREAD

**Issue by ddozza** [ansible-collections/hetzner.hcloud#29]
This is the error I receive when I try to create a floating IP.
```ERROR! couldn't resolve module/action 'hcloud_floating_ip'. This often indicates a misspelling, missing collection, or incorrect module path.

The error appears to be in '/home/davide/Projects/K8s/ansible/floating_ip.yml': line 9, column 7, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

  tasks:
    - name: Create a basic IPv4 Floating IP
      ^ here
```

The playbook looks like:

```
- name: Create Basic Server
  hosts: localhost
  connection: local
  gather_facts: False
  user: root
  vars:
    hcloud_token: xxxxxxxxxxxxxxxxx
  tasks:
    - name: Create a basic IPv4 Floating IP
      hcloud_floating_ip:
        api_token: "{{ hcloud_token }}"
        name: "my-floating-ip"
        home_location: "fsn1"
        type: ipv4
        state: present
      register: floatingIP 
```
Any suggestion?

**Comment by LKaemmerling** [comment]
This looks like a bug within your placebook. As you can see, it sees the syntax problem not within the hcloud_floating_ip module.  Maybe you could try it without a name of the task?

**Comment by ddozza** [comment]
As you suggested I tried without a name task. I get the same error:
```
Using /etc/ansible/ansible.cfg as config file
ERROR! couldn't resolve module/action 'hcloud_floating_ip'. This often indicates a misspelling, missing collection, or incorrect module path.

The error appears to be in '/home/davide/Projects/K8s/ansible/floating_ip.yml': line 9, column 7, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

  tasks:
    - name:
      ^ here
```
I can regularly use others plugins (hcloud_network, hcloud_subnetwork, hcloud_server, hcloud_server_network)  and it doesn't seem there is a syntax error in my playbook.

**Comment by LKaemmerling** [comment]
Do you use Ansible 2.10? Then you need to specify the full module name:
`hetzner.hcloud.hcloud_floating_ip`

**Comment by ddozza** [comment]
I'm using ansible 2.9

```
$ ansible --version
ansible 2.9.6
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/davide/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3/dist-packages/ansible
  executable location = /usr/bin/ansible
  python version = 3.8.2 (default, Jul 16 2020, 14:00:26) [GCC 9.3.0]
```

**Comment by LKaemmerling** [comment]
`hcloud_floating_ip` is not available in ansible 2.9 (as this is the last in-tree version). You should install the collection ( and/or update to Ansible 2.10 and use the collection)

**Comment by ddozza** [comment]
Solved!
